pages/dkp/konvoy/2.2/choose-infrastructure/eks/quick-start/index.md

```diff
@@ -131,7 +131,7 @@ Tips:
 
     <p class="message--note"><strong>NOTE: </strong>This kubeconfig is temporary and needs to be renewed either by running the above command with updated credentials and rerunning, or by running the AWS CLI command:</p>
 
-    ```yaml
+    ```bash
     aws eks --region us-west-2 update-kubeconfig --name default_<name-of-cluster>-control-plane
     ```
```

Comment:
Please remove this the note and the `aws eks` command here, it is no longer temporary.

Please also add a prereq to install https://docs.aws.amazon.com/eks/latest/userguide/install-aws-iam-authenticator.html